scipy/special/_add_newdocs.py

```diff
@@ -12988,8 +12988,41 @@ def add_newdoc(name, doc):
 
     Notes
     -----
-
+    
+    The log function used in the computation is the natural log.
+    
     .. versionadded:: 0.13.0
+    
+    Examples
+    --------
+    
+    Lets see how we can use the function to calculate the following sum: 
+    
+    .. math:: Z = \\sum_{j=0}^n x_j * log(y_j)
+    
+    using x and y as input arrays.
+    
+    >>> from scipy.special import xlogy
+    >>> import numpy as np
+    >>> Z = xlogy(np.array([1, 3, 5]), np.array([10, 30, 50]))
```

Comment:
Why these values? Can we instead use a known case? The result of the sum is not really appealing. And since there is no discussion on the values you took for `x,y` it's not clear.